Sources/TootSDK/TootClient/TootClient+Account.swift

```diff
@@ -282,6 +282,8 @@ extension TootFeature {
 
     /// Ability to edit your profile
     ///
+
+    @MainActor
```

Comment:
Any reason for this one? Might be problematic.

Reply:
Giving it a global actor to hang off - this could be changed to something else. Keen for suggestions here!

Reply:
I would leave it without any actor. I don't see why accessing it should be limited to main actor, or any other actor.

Reply:
I tried that and got no errors. Perhaps something changed since you added that annotation.